crates/re_sdk/src/web_viewer.rs

```diff
@@ -61,14 +65,47 @@ impl WebViewerSink {
         }
 
         Ok(Self {
+            open_browser,
             sender: rerun_tx,
-            _rerun_server: rerun_server,
+            rerun_server,
             _webviewer_server: webviewer_server,
         })
     }
 }
 
-/// Helper to spawn an instance of the [`re_web_viewer_server::WebViewerServer`].
+impl crate::sink::LogSink for WebViewerSink {
+    fn send(&self, msg: LogMsg) {
+        if let Err(err) = self.sender.send(msg) {
+            re_log::error_once!("Failed to send log message to web server: {err}");
+        }
+    }
+
+    #[inline]
+    fn flush_blocking(&self) {
+        if let Err(err) = self.sender.flush_blocking() {
+            re_log::error_once!("Failed to flush: {err}");
+        }
+    }
+}
+
+impl Drop for WebViewerSink {
+    fn drop(&mut self) {
+        if self.open_browser && self.rerun_server.num_accepted_clients() == 0 {
+            // For small scripts that execute fast we run the risk of finishing
+            // before the browser has a chance to connect.
+            // Let's give it a little more time:
+            std::thread::sleep(std::time::Duration::from_millis(1000));
```

Comment:
I think we should log an info message here already to avoid confusion about why rerun isn't shutting down. Let's say you called `serve` in future world of being able to also stream to a file: you might have done so just "because why not" and then be surprised about your small script taking longer